Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift

```diff
@@ -59,10 +61,17 @@ class DashboardViewModel: ObservableObject {
         guard let documentId = event.id else {
             fatalError("Event \(event.title) has no document ID.")
         }
-        try? await Firestore
-            .firestore()
-            .collection("maintenance_events")
-            .document(documentId)
-            .delete()
+        do{
+            try await Firestore
+                .firestore()
+                .collection("maintenance_events")
+                .document(documentId)
+                .delete()
+        }
+        catch{
+            showErrorAlert.toggle()
+            errorDeleting = error
+            //print("Error : \(error.localizedDescription)")
```

Comment:
Add a space between the curly brace and the words `do` and `catch`, and put the would catch on the line with the brace `}`
Also can you use the error's `localizedDescription`?

The statement would look like this
```swift
do {
} catch {
}
```